docs/container.md

```diff
@@ -171,6 +171,23 @@ printf('Container\'s name is $s', $container->getName());
 printf('Container\'s exit code is %d', $container->getExitCode());
 ```
 
+## Copy files or folders from a container
+
+File and folders can be downloaded as a tar file from the container. The copy() function allows a folder/file to be downloaded as tar stream and copyToDisk() saves the result as a tar file.
+In the following example the /etc/default folder from a ubuntu container called vanilla2 is downloaded to /tmp/resource.tar.
+
+```php
+<?php
+
+$lookfor = 'vanilla2';      
+$resource = '/etc/default'; 
```

Comment:
Actually, I would find the whole example more clear without the variables, ie:

```
$container = $manager->find('vanilla2');
$manager->copyToDisk($container, '/etc/default', '/tmp/file.tar');
```

What do you all think?